### PR TITLE
Separate profile runs in performance test

### PR DESCRIPTION
**Pull request by alexey-milovidov**
Changelog category (leave one):
- Build/Testing/Packaging Improvement

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Make performance test more stable and representative by splitting test runs and profile runs.

### CODE REVIEW

**Comment by alexey-milovidov** [comment]
```
Removing intermediate container bdbfff847862
Step 13/14 : COPY build.sh /
COPY failed: stat /place/sandbox-data/tasks/8/4/776403748/docker_root/tmp/docker-builder438288327/build.sh: no such file or directory
```

**Comment by alexey-milovidov** [comment]
@akuzm Could you please validate if new profile runs are collected to flame graphs?

**Comment by alexey-milovidov** [comment]
@akuzm If profile data does not work alright please make another PR.
At least now we can compare clang vs. gcc in fair way.

File: docker/test/performance-comparison/perf.py [hunk 1]
```diff
@@ -271,14 +267,17 @@ def substitute_parameters(query_templates, other_templates = []):
...
+            server_seconds += elapsed
+            print(f'query\t{query_index}\t{run_id}\t{conn_index}\t{elapsed}')
 
-            if c.last_query.elapsed > 10:
+            if elapsed > 10:
                 # Stop processing pathologically slow queries, to avoid timing out
                 # the entire test task. This shouldn't really happen, so we don't
                 # need much handling for this case and can just exit.
-                print(f'The query no. {query_index} is taking too long to run ({c.last_query.elapsed} s)', file=sys.stderr)
+                print(f'The query no. {query_index} is taking too long to run ({c.elapsed} s)', file=sys.stderr)
```
Comment (akuzm) [inline docker/test/performance-comparison/perf.py:280]: `c.elapsed` is a typo (I'll fix)

**Comment by akuzm** [comment]
> @akuzm Could you please validate if new profile runs are collected to flame graphs?

It's very simple to do: download the 'Test Output' and open the flame graph files. You would've noticed that they have about ten samples each (both CPU and Real variety), which means the setting didn't work and the queries use the default profiling period of 1 second.

**Comment by akuzm** [comment]
> > @akuzm Could you please validate if new profile runs are collected to flame graphs?
> 
> It's very simple to do: download the 'Test Output' and open the flame graph files. You would've noticed that they have about ten samples each (both CPU and Real variety), which means the setting didn't work and the queries use the default profiling period of 1 second.

On a second thought, this means that the flame graphs are for normal runs, and I'll need to add some other code to build them for profile runs. Will fix.

**Pull request merged**